release/release_logs/2.0.0/scalability/object_store.json

```diff
@@ -1,12 +1,12 @@
 {
-    "broadcast_time": 76.73496933199999,
+    "broadcast_time": 83.20190440599993,
     "num_nodes": 50,
     "object_size": 1073741824,
     "perf_metrics": [
         {
             "perf_metric_name": "time_to_broadcast_1073741824_bytes_to_50_nodes",
             "perf_metric_type": "LATENCY",
-            "perf_metric_value": 76.73496933199999
+            "perf_metric_value": 83.20190440599993
```

Comment:
minor regression - could be variance

Reply:
This seems to be variance in the test. https://b534fd88.us1a.app.preset.io/superset/explore/p/v5V9EpwA6yo/